internal/builders/docker/pkg/builder_test.go

```diff
@@ -15,13 +15,12 @@
 package pkg
 
 import (
-	"fmt"
 	"os"
-	"strings"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
 	intoto "github.com/in-toto/in-toto-golang/in_toto"
+	"github.com/slsa-framework/slsa-github-generator/internal/errors"
```

Comment:
Add a space so project imports aren't lumped in with 3rd party ones.

```suggestion
	"github.com/google/go-cmp/cmp"
	intoto "github.com/in-toto/in-toto-golang/in_toto"

	"github.com/slsa-framework/slsa-github-generator/internal/errors"
```

Reply:
Done.